### ISSUE TITLE
To be discussed: null coalesce rare false-positive

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1746]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | U4.0.6.4 |
| Language level | PHP 7.4 |

Current behaviour
--
Following #1667, just to be discussed.

There are a possible rare false-positive with (copy and paste from reply):

> I think that it could leads to a false-positive, but maybe rare. For instance: https://3v4l.org/UU5Jo
> 
> ```php
> $test = new stdClass;
> $test->code = null;
> 
> var_dump($test ? $test->code : 'N/A'); // null
> var_dump($test->code ?? 'N/A'); // N/A
> ```
> 
> So I suggest to not use it if the property could be nullable and `null` is in fact a possible result (and here I consider a rare situation).

In my suggestion, I think that it could be keep and the inspection could be ignored (`@noinspection`) in the rare cases where it could occurs.

**Comment by ea-inspections-team** [comment]
Thanks for reporting @rentalhost, the issue is fixed now.
